README.md

```diff
@@ -168,7 +168,7 @@ You are free to write your own, which requires you to implement the following in
 public interface Selector<A> {
 
     @Nullable
-    Binding<A> select(final ClientHttpResponse response, final Collection<Binding<A>> bindings) throws IOException;
+    Binding<A> select(final ClientHttpResponse response, final Map<A, Binding<A>> bindings) throws IOException;
```

Comment:
For bindings you are right, but for Routers and Selectors it is more natural thing to think of maps between attributes and and executors. It would be even possible to remove the Bindings here. In addition this is an internal change not visible in the API.